src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,70 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Collections;
+
 public class WorkWithFile {
+    private static final String COMMA_DELIMITER = ",";
+    private static final String LINE_DELIMITER = System.lineSeparator();
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String report = formReport(readFile(fromFileName));
+        writeToFile(toFileName, report);
+    }
```

Comment:
The method 'getStatistic' is doing more than one task: reading from a file, processing data, and writing to a file. According to the checklist, these should be separate operations. Consider refactoring to adhere to the single responsibility principle.